### PR TITLE
i#3544 RV64: Add docs on cross-compiling for RISCV64 on Linux

### PR DESCRIPTION
**Pull request by chenhy0106**
1. Add docs on cross-compiling building commands for RISCV64 on Linux.
2. Fix some typos.

Issue: #3544

### CODE REVIEW

File: .github/workflows/ci-riscv64.yml
Comment (ksco) [inline .github/workflows/ci-riscv64.yml:1]: To the PR description: it's `Issue: #3544` not `i#3544`.

**Review by ksco — APPROVED** [review]

File: .github/workflows/ci-riscv64.yml
Comment (ksco) [reply .github/workflows/ci-riscv64.yml:1]: And remove the period in the PR title.

File: .github/workflows/ci-riscv64.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ jobs:
     # Install cross-compiler for cross-compiling Linux build.
     # Unfortunately there are no libunwind or compression cross-compile
     # packages so we unpack the native versions and copy their files.
-    # Unfortunately, only kinetic (ubuntu 22.10) has linunwind-riscv64 packages.
+    # Unfortunately, only kinetic (ubuntu 22.10) has libunwind-riscv64 packages.
     # I prefer debian-ports here, it is more close to upstream.
```
Comment (derekbruening) [inline .github/workflows/ci-riscv64.yml:74]: s/more close/closer/

File: api/docs/building.dox [hunk 2]
```diff
@@ -247,6 +247,42 @@ To build a client, again use the toolchain file, as well as pointing at a Dynamo
...
 
+## Cross-Compiling for 64-bit RISC-V (RISCV64) on Linux
+
+Prepare cross-compiling environment for RISCV64:
+
+```
+$ sudo apt-get -y install doxygen vera++ cmake crossbuild-essential-riscv64 git
+```
+
+Some packages may need to be manually installed for cross-compiling environment. You can also refer to our [CI workflow file](https://github.com/DynamoRIO/dynamorio/blob/master/.github/workflows/ci-riscv64.yml) for creating RISCV64 build environment.
```
Comment (derekbruening) [inline api/docs/building.dox:258]: s/creating/creating a/

**Review by derekbruening — APPROVED** [review]

**Pull request merged**